packages/office-addin-debugging/src/start.ts

```diff
@@ -306,7 +306,18 @@ export async function startDebugging(manifestPath: string, options: StartDebuggi
     // enable loopback for Edge
     if (isWindowsPlatform && parseInt(os.release(), 10) === 10) {
       const name = isDesktopAppType ? "EdgeWebView" : "EdgeWebBrowser";
-      await devSettings.ensureLoopbackIsEnabled(name);
+      try {
+        await devSettings.ensureLoopbackIsEnabled(name);
+      } catch (err: any) {
+        // if add loopback exemption failed, report the error then continue
+        console.error(err)
+        console.warn("Failed to add loopback exemption.\nTry to sideload the Office Add-in without loopback exemption.\nThe Office Add-in may not load correctly.\n")
```

Comment:
Just a slight change to the message . . . "Failed to add loopback exemption.\nWill try to sideload the Office Add-in without the loopback exemption, but it might not load correctly from localhost.\n"